.taprc

```diff
@@ -1 +1 @@
-check-coverage: false
+100: true
```

Comment:
this is the default, you can just remove the .taprc file

Reply:
done